src/condition.cpp

```diff
@@ -1734,3 +1733,54 @@ bool ConditionSpellGroupCooldown::startCondition(Creature* creature)
 	}
 	return true;
 }
+
+bool ConditionDrunk::startCondition(Creature* creature)
+{
+	if (!Condition::startCondition(creature)) {
+		return false;
+	}
+
+	creature->setDrunkness(drunkness);
```

Comment:
every single occurence of drunkness rename to drunkenes, drunkness is incorrect

Reply:
Actually, I double checked before renaming and the correct spelling is: drunkenness. :p.